platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -204,11 +204,11 @@ public long getTransitionDuration() {
   /**
    * Set the animation duration for style changes.
    *
-   * @param duration Duration in seconds
+   * @param durationMs Duration in milliseconds
    */
   @UiThread
-  public void setTransitionDuration(long duration) {
```

Comment:
Ivo mentioned the same on chat regards using Duration from joda.time, will ticket it out.